apps/src/music/views/PackDialog.tsx

```diff
@@ -33,23 +40,35 @@ const PackEntry: React.FunctionComponent<PackEntryProps> = ({
   const isPlayingPreview = previewSound && playingPreview === soundPath;
   const imageSrc = library?.getPackImageUrl(folder.id);
 
+  const onEntryClick = useCallback(() => {
+    onSelect(folder);
+
+    if (usePackDialogPreview) {
+      if (soundPath && !isPlayingPreview) {
+        onPreview(soundPath);
+      }
+    }
+  }, [folder, isPlayingPreview, onPreview, onSelect, soundPath]);
+
   const onPreviewClick = useCallback(
     (e: Event) => {
-      if (soundPath && !isPlayingPreview) {
+      if (isPlayingPreview) {
+        onStopPreview();
+      } else if (soundPath && !isPlayingPreview) {
```

Comment:
just noting that `!isPlayingPreview` is unneeded and redundant - since if it were true, it would've been caught by the prior conditional.

That said, it might be useful to keep in if it adds clarity.